demo/demo.jsx

```diff
@@ -2,7 +2,7 @@
 import _ from "lodash";
 import React from "react";
 import {VictoryBar} from "../src/index";
-import {VictoryChart} from "victory-chart";
+// import {VictoryChart} from "victory-chart";
```

Comment:
yeah, I haven't updated `victory-chart` to work with `react-art` yet, so that is all temporarily broken